ThirdPartyPrivileged/IDCACnano.txt

```diff
@@ -0,0 +1,103 @@
+! Title: I don't care about cookies Nano expansion
+! Version: 20May2019v1
+! Expires: 7 days
+! Homepage: https://github.com/DandelionSprout/adfilt
+! Support: https://github.com/DandelionSprout/adfilt/issues
+! License: GPL-3.0
+! Description: This is an unofficial adaption to Nano Adblocker only, of certain entries that are otherwise not present in the official adblocker version of I Don't Care About Cookies, but which are part of the extension version's 'common6.js' file.
+!
+! For the homepage of I Don't Care About Cookies, visit https://www.i-dont-care-about-cookies.eu/
+! For details about the lists that the lead adaptor (Dandelion Sprout) has made, visit https://github.com/DandelionSprout/adfilt/blob/master/Wiki/General-info.md#english
+! For the homepage of the repository where this list is hosted, visit https://github.com/NanoAdblocker/NanoFilters
+!
+! In case you haven't done so already, make sure to subscribe to the official IDCAC list at https://www.kiboke-studio.hr/i-dont-care-about-cookies/abp/ as well.
+!
+! Important note: The "nanop-easy-set-cookie.js" entries are known to sometimes not get along with the official I Don't Care About Cookies extension, such as on Fandom.com. Therefore you should only use either this list or the extension.
+
+aceandtate.com##+js(nanop-easy-set-cookie.js, cookieconsent_status=allow)
+agroneo.com##+js(nanop-easy-set-cookie.js, consent=yes)
+alletop10lijstjes.nl,onmsft.com##+js(nanop-easy-set-cookie.js, euconsent=1)
+antyweb.pl##+js(nanop-easy-set-cookie.js, aw-privacy-approval=true)
+appi.org##+js(nanop-easy-set-cookie.js, apa_gdpr_accepted=True)
+archief.amsterdam##+js(nanop-easy-set-cookie.js, verklaring=1)
+archiveofourown.org##+js(nanop-easy-set-cookie.js, accepted_tos=20180523)
+assocarabinieri.it##+js(nanop-easy-set-cookie.js, cookieLaw=1)
+betterhelp.com##+js(nanop-easy-set-cookie.js, gdpr_cookie_consent_given=yes)
+choosist.com##+js(nanop-easy-set-cookie.js, accept_cookie=1)
+cindicator.com##+js(nanop-easy-set-cookie.js, accepted_cookie=true)
+cjponyparts.com##+js(nanop-easy-set-cookie.js, privacypopup=stop)
+cmore.se##+js(nanop-easy-set-cookie.js, cookieBannerDismissed=true)
+curiositystream.com##+js(nanop-easy-set-cookie.js, gdpr_opt_out=0)
+datanyze.com##+js(nanop-easy-set-cookie.js, CookieUsage=true)
+depop.com##+js(nanop-easy-set-cookie.js, bisc__ack=1)
+dlmirror.eu##+js(nanop-easy-set-cookie.js, consent=1)
+doc.fr##+js(nanop-easy-set-cookie.js, cookieAgreement=True)
+duolingo.com##+js(nanop-easy-set-cookie.js, gdpr_cookie=true)
+dwell.com##+js(nanop-easy-set-cookie.js, gdpr=1)
+eneba.com##+js(nanop-easy-set-cookie.js, GDPR=1)
+enviam.de##+js(nanop-easy-set-cookie.js, dws01-level=2)
+esea.net##+js(nanop-easy-set-cookie.js, cookie_consent=1)
+euroclear.com##+js(nanop-easy-set-cookie.js, cc_necessary=true', 'cc_functional=true)
```

Comment:
Is this generated? Your parser needs some work. You need two lines to inject two cookies.

Reply:
Ah, I didn't know that such a limitation applied to `nanop-easy-set-cookie.js`. I'll fix it in about half an hour; I just need to finish some sorting work on Pixiv first.

Reply:
I've fixed it now. For the time being I'm converting the entries manually, since there's relatively few of them.